test/request.test.ts

```diff
@@ -40,6 +40,34 @@ describe('Request', () => {
       expect(req).toBeInstanceOf(global.Request)
       expect(req.url).toBe('http://localhost/foo.txt')
     })
+
+    it('should generate only one `AbortController` per `Request` object created', async () => {
```

Comment:
Just a sanity check to make sure we do not create a new AbortController every time we access it, or any other property since it calls `getRequestCache`.